framework/base/ErrorHandler.php

```diff
@@ -264,6 +281,13 @@ public function handleFatalError()
     {
         unset($this->_memoryReserve);
 
+        if (isset($this->_workingDirector)) {
+            // fix working directory for some Web servers e.g. Apache
+            chdir($this->_workingDirectory);
+            // flush memory
+            unset($this->_workingDirector);
```

Comment:
```suggestion
            $this->_workingDirectory = null;
```

unset will actually remove the property from the class, which is different from setting it to `null`.